src/EventTimer.spec.ts

```diff
@@ -6,27 +6,27 @@ jest.mock('./GoogleAnalytics', () => ({
 }));
 
 describe('EventTimer', () => {
-	const performance = {
-		now: jest.fn(),
-		mark: jest.fn(),
-		getEntriesByName: jest.fn().mockReturnValue([
-			{
-				duration: 1,
-				entryType: 'mark',
-				name: 'commercial event',
-				startTime: 1,
-			},
-		]),
-	};
-
-	Object.defineProperty(window, 'performance', {
-		configurable: true,
-		enumerable: true,
-		value: performance,
-		writable: true,
-	});
-
 	beforeEach(() => {
+		const performance = {
+			now: jest.fn(),
+			mark: jest.fn(),
+			getEntriesByName: jest.fn().mockReturnValue([
+				{
+					duration: 1,
+					entryType: 'mark',
+					name: 'commercial event',
+					startTime: 1,
+				},
+			]),
+		};
```

Comment:
This `const performance` can still live outside of the `beforeEach`. You don’t need to redefine this variable each time, only assign to `window.performance`.

*not a blocker*